content/en/logs/guide/access-your-log-data-programmatically.md

```diff
@@ -0,0 +1,524 @@
+    }
+}
+
+```
+
+### Limit the number of results retrieved
+
+With the following API call, limit the number of log events retrieved. The `limit` indicates the maximum number of log events returned in the response. The maximum limit is `1000`.
+
+```bash
+
+curl -L -X POST "https://api.{{< region-param key="dd_site" code="true" >}}/api/v2/logs/events/search" -H "Content-Type: application/json" -H "DD-API-KEY: <DATADOG_API_KEY>" -H "DD-APPLICATION-KEY: <DATADOG_APP_KEY>" --data-raw '{
+  "filter": {
+    "from": "2020-10-07T00:00:00+00:00",
+    "to": "2020-10-07T00:15:00+00:00",
+    "query": "*"
+  },
+   "page": {
+    "limit":2
+  },
+  "sort":"-@pageViews"
+}'
+
+```
+
+**Response:**
+
+```json
+
+{
+    "meta": {
+        "page": {
+            "after": "eyJhZnRlciI6IkFRQUFBWFVBWFZOU3Z1TXZXd0FBQUFCQldGVkJXRlpPVTJJMlpXY3hYM2MyTFZWQlFRIiwidmFsdWVzIjpbIjUwMCJdfQ"
+        }
+    },
+    "data": [
+        {
+            "attributes": {
+                "status": "info",
+                "service": "pageViewService",
+                "tags": [
+                    "source:postman",
+                    "project:test"
+                ],
+                "timestamp": "2020-10-07T00:01:41.909Z",
+                "host": "my.sample.host",
+                "attributes": {
+                    "hostname": "my.sample.host",
+                    "pageViews": "700",
+                    "user": "steve",
+                    "service": "pageViewService"
+                },
+                "message": "Sample message"
+            },
+            "type": "log",
+            "id": "AQAAAXUAXRYVJGxvDQAAAABBWFVBWFJZVmd2ZlktbUdUZjRBQQ"
+        },
+        {
+            "attributes": {
+                "status": "info",
+                "service": "pageViewService",
+                "tags": [
+                    "source:postman",
+                    "project:test"
+                ],
+                "timestamp": "2020-10-07T00:01:57.586Z",
+                "host": "my.sample.host",
+                "attributes": {
+                    "hostname": "my.sample.host",
+                    "pageViews": "500",
+                    "user": "bob",
+                    "service": "pageViewService"
+                },
+                "message": "Sample message"
+            },
+            "type": "log",
+            "id": "AQAAAXUAXVNSvuMvWwAAAABBWFVBWFZOU2I2ZWcxX3c2LVVBQQ"
+        }
+    ],
+    "links": {
+        "next": "https://api.datadoghq.com/api/v2/logs/events?sort=-%40pageViews&filter%5Bto%5D=2020-10-07T00%3A15%3A00%2B00%3A00&page%5Bcursor%5D=eyJhZnRlciI6IkFRQUFBWFVBWFZOU3Z1TXZXd0FBQUFCQldGVkJXRlpPVTJJMlpXY3hYM2MyTFZWQlFRIiwidmFsdWVzIjpbIjUwMCJdfQ&filter%5Bfrom%5D=2020-10-07T00%3A00%3A00%2B00%3A00&filter%5Bquery%5D=%2A&page%5Blimit%5D=2"
+    }
+}
+
+```
+
+### Pagination
+
```

Comment:
Let's mention somewhere that we discourage the use of relative timerange when using pagination. Why? Because your pagination cursor can move in time slower than the actual time.